crMeta/crMetadata.go

```diff
@@ -157,22 +159,24 @@ func (doc *TargetDocument) GetMetadata() (*CRMetadata, error) {
 		if err != nil {
 			return nil, err
 		}
-		cas, cvCas, cvSrc, cvVer, pvMap, mvMap, err := getMetaFromMCResponse(doc.resp)
-		if err != nil {
-			return nil, err
-		}
 		meta := CRMetadata{
 			docMeta: &docMeta,
 		}
-		if len(pvMap) > 0 {
-			meta.hadPv = true
-		}
-		if len(mvMap) > 0 {
-			meta.hadMv = true
-		}
-		meta.hlv, err = hlv.NewHLV(doc.source, cas, cvCas, cvSrc, cvVer, pvMap, mvMap)
-		if err != nil {
-			return nil, err
+		if doc.includeHlv {
+			cas, cvCas, cvSrc, cvVer, pvMap, mvMap, err := getHlvFromMCResponse(doc.resp)
+			if err != nil {
+				return nil, err
+			}
+			if len(pvMap) > 0 {
+				meta.hadPv = true
```

Comment:
I added comment in the code. pvMap/mvMap here come straight from document XATTR, and hadPv/hadMv just record that fact. We need this info to decide whether to send delete for the MV/PV path in the subdoc command. The updated HLV is in meta.hlv and it will decide whether we still have it.